assets/plugin-template/android/build.gradle

```diff
@@ -11,7 +11,7 @@ buildscript {
         mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:7.2.1'
+        classpath 'com.android.tools.build:gradle:7.2.2'
```

Comment:
Is this the gradle version we want to pin? Or should it be one of the 7.4.1 ones?